tests/ikev2-weak-dh/test.yaml

```diff
@@ -31,3 +32,38 @@ checks:
         ikev2.payload[1]: KeyExchange
         ikev2.payload[2]: SecurityAssociation
         ikev2.payload[3]: NoNextPayload
+
+  - filter:
+      count: 1
+      version: 5
+      match:
+        event_type: ikev2
+        ikev2.version_major: 2
+        ikev2.exchange_type: 34
+        ikev2.message_id: 0
+        ikev2.init_spi: "61d3693ce12af528"
+        ikev2.resp_spi: "0000000000000000"
+        ikev2.role: initiator
+        ikev2.errors: 0
+        ikev2.payload[0]: Nonce
+        ikev2.payload[1]: KeyExchange
+        ikev2.payload[2]: SecurityAssociation
+        ikev2.payload[3]: NoNextPayload
+
+  - filter:
+      count: 1
+      min-version: 6
+      match:
+        event_type: ike
```

Comment:
@fhonza Can you add a simple comment to say that the only change is renaming the event from ikev2 to generic ike ?

Reply:
yes, will add one...